jenkins/sign-artifacts/sign-standalone-artifacts.jenkinsfile

```diff
@@ -19,15 +19,17 @@ pipeline {
                 trim: true,
                 description: 'Path to upload to artifacts and signatures on s3. Eg: dummy_project/1.0'
         )
-        choice(
-                choices: ['linux'],
+        string(
+                defaultValue: 'linux',
                 name: 'DISTRIBUTION_PLATFORM',
+                trim: true,
                 description: 'What platform is this distribution build for?'
         )
-        choice(
-                choices: ['.sig'],
+        string(
```

Comment:
I don't think we should change this to string as it would allow user to enter any value for creating signature. It needs to be from a fixed set of list. The sole reason of dropdown menus is to prevent this.